cmd/clusterctl/client/init.go

```diff
@@ -73,6 +73,9 @@ type InitOptions struct {
 	// IgnoreValidationErrors allows for skipping the validation of provider installs.
 	// NOTE this should only be used for development
 	IgnoreValidationErrors bool
+
+	// allowMissingProviderCRD allows listing of images if the provider CRD is missing
```

Comment:
```suggestion
	// allowMissingProviderCRD allows listing of images if the provider CRD is missing.
```
nit For consistency.